pkg/apis/istio/v1alpha2/values/fix_values_structs.patch

```diff
@@ -0,0 +1,224 @@
+ 	Hub     string `protobuf:"bytes,2,opt,name=hub,proto3" json:"hub,omitempty"`
+-	// 	NodeSelector map[string]interface{} `json:"nodeSelector"`
++	NodeSelector map[string]interface{} `json:"nodeSelector"`
+ 	Resources            *ResourcesConfig `protobuf:"bytes,3,opt,name=resources,proto3" json:"resources,omitempty"`
+ 	Tag                  string           `protobuf:"bytes,4,opt,name=tag,proto3" json:"tag,omitempty"`
+ 	XXX_NoUnkeyedLiteral struct{}         `json:"-"`
+@@ -291,7 +292,7 @@ type CoreDNSConfig struct {
+ 	CoreDNSImage       string `protobuf:"bytes,1,opt,name=coreDNSImage,proto3" json:"coreDNSImage,omitempty"`
+ 	CoreDNSPluginImage string `protobuf:"bytes,2,opt,name=coreDNSPluginImage,proto3" json:"coreDNSPluginImage,omitempty"`
+ 	Enabled            bool   `protobuf:"varint,3,opt,name=enabled,proto3" json:"enabled,omitempty"`
+-	// 	NodeSelector       map[string]interface{} `json:"nodeSelector,omitempty"`
++	NodeSelector       map[string]interface{} `json:"nodeSelector,omitempty"`
+ 	ReplicaCount         uint32   `protobuf:"varint,4,opt,name=replicaCount,proto3" json:"replicaCount,omitempty"`
+ 	XXX_NoUnkeyedLiteral struct{} `json:"-"`
+ 	XXX_unrecognized     []byte   `json:"-"`
+@@ -445,14 +446,14 @@ type EgressGatewayConfig struct {
+ 	Env              map[string]string           `protobuf:"bytes,8,rep,name=env,proto3" json:"env,omitempty" protobuf_key:"bytes,1,opt,name=key,proto3" protobuf_val:"bytes,2,opt,name=value,proto3"`
+ 	Labels           *GatewayLabelsConfig        `protobuf:"bytes,9,opt,name=labels,proto3" json:"labels,omitempty"`
+ 	NodeSelector     map[string]string           `protobuf:"bytes,10,rep,name=nodeSelector,proto3" json:"nodeSelector,omitempty" protobuf_key:"bytes,1,opt,name=key,proto3" protobuf_val:"bytes,2,opt,name=value,proto3"`
+-	// 	PodAnnotations                   map[string]interface{}      `json:"podAnnotations,omitempty"`
+-	// 	PodAntiAffinityLabelSelector     []map[string]interface{}    `json:"podAntiAffinityLabelSelector"`
+-	// 	PodAntiAffinityTermLabelSelector []map[string]interface{}    `json:"podAntiAffinityTermLabelSelector"`
++	PodAnnotations                   map[string]interface{}      `json:"podAnnotations,omitempty"`
++	PodAntiAffinityLabelSelector     []map[string]interface{}    `json:"podAntiAffinityLabelSelector"`
++	PodAntiAffinityTermLabelSelector []map[string]interface{}    `json:"podAntiAffinityTermLabelSelector"`
+ 	Ports         []*PortsConfig   `protobuf:"bytes,11,rep,name=ports,proto3" json:"ports,omitempty"`
+ 	Resources     *ResourcesConfig `protobuf:"bytes,12,opt,name=resources,proto3" json:"resources,omitempty"`
+ 	SecretVolumes []*SecretVolume  `protobuf:"bytes,13,rep,name=secretVolumes,proto3" json:"secretVolumes,omitempty"`
+-	// 	ServiceAnnotations               map[string]interface{}      `json:"serviceAnnotations,omitempty"`
+-	//  Type                             corev1.ServiceType          `json:"type,omitempty"`
```

Comment:
why is Type commented out?

Reply:
The go-to-proto tool cannot recognize the type imported so I have to comment out first